server/repository/userRole/update.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseProtocol, DB } from '@server/db'
+import { UserRole, RoleName } from '@meta/user'
+import { Objects } from '@core/utils'
+
+export const update = async (
+  props: {
+    userRole: UserRole<RoleName, any>
+    accept?: boolean
```

Comment:
do we have a case where we update the user role without accepting ? I only see the accepted column updated.
Please, call this method 

```acceptInvitation(props:{userRole}) ```